src/waypoint.jsx

```diff
@@ -333,13 +352,30 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
+    if (this.props.noWrapper) {
```

Comment:
Yes, but someone who uses `this.props.noWrapper` without actually passing children should receive an error message – which will happen thanks to `React.Children.only`.